src/user.ts

```diff
@@ -16,20 +18,37 @@ class User {
     }
 
     createAccount(accountName: string, masterPassword: string) {
-        const { privateKey, salt } = this.generatePrivateKeyFromPassword(masterPassword);
-
-        const passwordPublicKey = this.authenticator.storeKey(AuthenticatorLevel.Password, privateKey, masterPassword);
-
-        this.id.create(accountName, passwordPublicKey, salt);
+        console.log(accountName, masterPassword);
     }
 
-    generatePrivateKeyFromPassword(password: string) {
+
+    async generatePrivateKeyFromPassword(password: string): Promise<{ privateKey: PrivateKey, salt: Buffer }> {
```

Comment:
why are there two functions? generatePrivateKeyFromPassword and generateRandomPrivateKey?

Reply:
one that generates privateKey from password
and the other generates a random Key using the same algorithm

Reply:
also both functions are in the design dooc https://www.notion.so/tonomy-foundation/2022-08-03-Tonomy-Create-Account-feature-Design-doc-9f270a2dedbd4991abbbac2226e16d07